src/Type/Doctrine/Descriptors/FloatType.php

```diff
@@ -26,7 +39,38 @@ public function getWritableToDatabaseType(): Type
 
 	public function getDatabaseInternalType(): Type
 	{
-		return TypeCombinator::union(new \PHPStan\Type\FloatType(), new IntegerType());
```

Comment:
I tried hard finding at least single case where integer is returned by some driver (in #506), but was not able to find any. So I believe it was just wrong, int is probably never returned.

On the other hand, numeric-string is returned pretty often and was missing here.

Reply:
I'll separate this to another PR.